src/player.ts

```diff
@@ -126,29 +124,32 @@ export class Player {
   }
 
   /**
-   * helper to create agent pub keys and install multiple apps for scenario initialization
+   * helper to create agent keys and install multiple apps for scenario initialization
    */
   installAgentsHapps = async (agentsHapps: InstallAgentsHapps): Promise<InstalledAgentHapps> => {
     this._conductorGuard(`Player.installHapps`)
-    const admin: AdminWebsocket = this._conductor!.adminClient!
     return await Promise.all(agentsHapps.map(async agentHapps => {
-      let agentPubKey = await admin.generateAgentPubKey()
```

Comment:
@zippy 
per our design, you can't take the agentPubKey generation out of this level, because you wanted the same agentPubKey being able to re-used across hApps remember? :)

Reply:
duh!  Good catch.  I'll fix.